adagrams/game.py

```diff
@@ -1,13 +1,119 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+LETTER_SCORE = {
+    "A": 1,
+    "E": 1,
+    "I": 1,
+    "O": 1,
+    "U": 1,
+    "L": 1,
+    "N": 1,
+    "R": 1,
+    "S": 1, 
+    "T": 1,
+    "D": 2, 
+    "G": 2,
+    "B": 3, 
+    "C": 3, 
+    "M": 3, 
+    "P": 3,
+    "F": 4, 
+    "H": 4, 
+    "V": 4, 
+    "W": 4, 
+    "Y": 4,
+    "K": 5,
+    "J": 8, 
+    "X": 8,
+    "Q": 10,
+    "Z": 10
+}
+
 def draw_letters():
-    pass
+    all_letters = []
+    hand = []
+
+    for letter, count in LETTER_POOL.items():
+        for _ in range(0, count):
+            all_letters.append(letter)
```

Comment:
This works and I love the underscore notation you used to indicate that the value there doesn't matter and won't be used within the body of the loop! This nested for loop works, but we could also try and avoid that inner loop by using the `extend()` function! There were a couple examples of this in the wrap up today that looked like the following: 

```suggestion
        all_letters.extend([letter] * count)
```

This starts by creating a list that consists of the current letter `count` times (`A: 9` becomes `['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A']`). It then extends `all_letters` to contain everything from the list that was just created!

At the end of the day, both approaches are equal in terms of time and space complexity, so choosing one over the other is a matter of personal preference. Typically though, if we can avoid using a nested loop, we should try to do so!